pages/signup.php

```diff
@@ -140,14 +175,15 @@
       <input type="text" class="my-2" name="uname" placeholder="Username" class="requiredField">
       <input type="text" class="my-2" name="email" placeholder="Email" class="requiredField">
       <input type="number" class="my-2" name="hosp_id" placeholder="Hospital ID" class="requiredField">
-      <p class="text-start text-light m-0 p-0">Enter hospital Id given by hospital</p>
+      <!-- <p class="text-start text-light m-0 p-0">Enter hospital Id given by hospital</p> -->
```

Comment:
Why is this commented? This field is required during signup.

Reply:
> Why is this commented? This field is required during signup.

I commented out or omitted this field since it was comming in between the input email area and the hospital id area , since there's already an input box where the user can input the hospital id and also it looked odd and also in the main website or the hosted website there was no such thing so I decided to omit it . Anyway if you want I can add the line in placeholder though.